fix(gatsby-admin): Setup Gatsby Admin site (#23291)
gatsbyjs/gatsby: packages/gatsby-admin/README.md

# Gatsby Admin

A visual interface to configure your Gatsby site.

> We have not packaged this nicely yet, so its not installable

Reviewer: ```suggestion
We have not packaged this nicely yet, so it is not installable.
```

# Gatsby Admin

A visual interface to configure your Gatsby site.

> We have not packaged this nicely yet, so it is not installable.

Reviewer 2: Your post on Slack indicated that it's now possible. So is this info outdated or that feature in another PR/branch?

Author: In another PR (#23368), but @freiksenet pointed out that we don't actually want a plugin as we have to hook into the parent process—so I have to finish #22600 (which requires finishing #22759, which is what I'm working on right now) first 😬

## How to develop it

However, you can do some manual set up in order to work with it locally. Follow these steps:

1. Navigate to the monorepo and open the `packages/gatsby-admin` directory.
2. In that directory, run `yarn develop`.

>    > If you see eslint errors you'll need to temporarly replace all references to `___loader` with `window.___loader`.

Reviewer: ```suggestion
   > If you see eslint errors you'll need to temporarly replace all references to `___loader` with `window.___loader` inside of `gatsby-link/index.js`.
```

>    > If you see eslint errors you'll need to temporarly replace all references to `___loader` with `window.___loader` inside of `gatsby-link/index.js`.

Reviewer 3: ```suggestion
   > If you see eslint errors you'll need to temporarily replace all references to `___loader` with `window.___loader` inside of `gatsby-link/index.js`.
```
